libs/utils/logWebVitals.js

```diff
@@ -0,0 +1,97 @@
+const LANA_CLIENT_ID = 'pageperf';
+
+let lanaSent = false;
+
+function sendToLana(lanaData) {
+  if (lanaSent) return;
+  const ua = window.navigator.userAgent;
+
+  Object.assign(lanaData, {
+    chromeVer: (ua.match(/Chrome\/(\d+\.\d+\.\d+\.\d+)/) || [])[1] || '',
+    country: sessionStorage.getItem('akamai') || '',
+    // eslint-disable-next-line compat/compat
+    downlink: window.navigator?.connection?.downlink || '',
+    loggedIn: window.adobeIMS?.isSignedInUser() || false,
+    os: (ua.match(/Windows/) && 'win')
+      || (ua.match(/Mac/) && 'mac')
+      || (ua.match(/Android/) && 'android')
+      || (ua.match(/Linux/) && 'linux')
+      || '',
+    windowHeight: window.innerHeight,
+    windowWidth: window.innerWidth,
+    url: `${window.location.host}${window.location.pathname}`,
+  });
+
+  lanaData.cls ||= 0;
+  const lanaDataStr = Object.entries(lanaData)
+    .sort(([a], [b]) => a.localeCompare(b))
+    .map(([key, value]) => `${key}=${value}`)
+    .join(',');
+
+  window.lana.log(lanaDataStr, {
+    clientId: LANA_CLIENT_ID,
+    sampleRate: 100,
+  });
+
+  lanaSent = true;
+}
+
+function observeCLS(lanaData) {
+  let cls = 0;
+  /* c8 ignore start */
+  new PerformanceObserver((entryList) => {
+    for (const entry of entryList.getEntries()) {
+      if (!entry.hadRecentInput) {
+        cls += entry.value;
+      }
```

Comment:
Nit: this could be an one liner.
```suggestion
      if (!entry.hadRecentInput) cls += entry.value;
```